src/components/SourceFooter.css

```diff
@@ -12,50 +12,39 @@
   user-select: none;
 }
 
-html:not([dir="rtl"]) .source-footer .command-bar {
-  float: right;
+.source-footer__command-bar {
```

Comment:
I'm personally a fan of BEM for a number of reasons mentioned (including perf) and I'd like to see it used more broadly in the project and across the rest of the devtools.  However unless we decide to use it more broadly this one off case creates an inconsistency that I don't think we want.